modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,504 @@
+[[filter-by-time]]
+=== Filter Data by Time Period
+
+Use the time period selector (<<dashboard-overview, item 6>> in the example) to display data points collected for a given time period (such as the last 15 minutes or the last 24 hours). Your setting applies across all data in *Anypoint Insights* overview, including the *Entities List*.
+
+If you don't see data on the page, check that the selected time range is correct.
+
+//
+//H2: CHANGE ENVIRONMENT
+[[change-environment]]
+== Change the Environment
+
+Charts in Anypoint Insights provide metrics on a specific environment (such as Production or Sandbox). 
+
+To change current or default environment:
+
+. Open Anypoint Monitoring. 
+. Set a new environment:
+
+* To select an environment without changing the default, use the environment selector in Anypoint Monitoring, for example:
++
+image::insights-environment-selector.png["Environment selector in Anypoint Monitoring"]
+* To select a default environment:
+
+.. Click your profile icon (see <<profile, item 2>> in the example). 
+.. Select the name of your profile to open Access Management *Settings*.
+.. In *Settings*, select an environment as the default environment to view for the selected business group in Anypoint Monitoring, for example: 
++
+image::insights-default-selector.png["Default selector in Access Management"]
+.. Return to Anypoint Monitoring to check for the new business group setting.
++
+Refresh the page, if necessary. 
+
+//
+//H2: CHANGE BUSINESS GROUP
+[[change-bg]]
+== Change the Business Group
+
+Charts in Anypoint Insights display metrics for deployments in a specific business group. If you belong to more than one business group, you can change the selected group and the business group that Anypoint Monitoring selects by default.
+
+//do not change the order of numbers in the graphic or calloutlist
+[[profile]]
+image::insights-business-group-change.png["Business group selection options"]
+
+[calloutlist]
+. Business group selector 
+. User profile, for selecting defaults
+
+To change the business group setting: 
+
+. Open Anypoint Monitoring. 
+. Set a new business group:
+
+* To select a business group without changing the default, use the business group selector in Anypoint Insights. 
+* To select a default business group:
+
+.. Click your profile icon (see <<profile, item 2>> in the example). 
+.. Select the name of your profile to open Access Management *Settings*.
+.. In Access Management *Settings*, select a business group to use as the default, for example:
++
+image::insights-default-selector.png["Default selector in Access Management"]
+.. Return to Anypoint Monitoring to check for the new default setting.
++
+Refresh the page, if necessary. 
+
+[[download-chart-data]]
+== Download Chart Data
+
+After opening a set of charts in Anypoint Insights, you can download metrics from charts that provide a download menu in the chart.
+
+. Click the vertical ellipsis on a chart to open the download menu. 
```

Comment:
Good place for an actual icon, maybe?

Reply:
Oh yeah, I creaeted one that's used elsewhere. Adding. 
TODO: @dukesphere

Reply:
```suggestion
. Click the image:more-menu-vert-ellipsis.png["",15,15] (*More menu*)) on a chart to open the download menu. 
```